flink-end-to-end-tests/test-scripts/test_batch_wordcount.sh

```diff
@@ -52,13 +78,11 @@ case $INPUT_TYPE in
     ;;
 esac
 
-OUTPUT_LOCATION="${TEST_DATA_DIR}/out/wc_out"
-
-mkdir -p "${TEST_DATA_DIR}"
-
+mkdir -p "$(dirname $OUTPUT_PATH)"
 start_cluster
 
 # The test may run against different source types.
 # But the sources should provide the same test data, so the checksum stays the same for all tests.
-eval "${FLINK_DIR}/bin/flink run -p 1 ${FLINK_DIR}/examples/batch/WordCount.jar ${INPUT_ARGS} --output ${OUTPUT_LOCATION}"
-check_result_hash "WordCount (${INPUT_TYPE})" "${OUTPUT_LOCATION}" "${RESULT_HASH}"
+${FLINK_DIR}/bin/flink run -p 1 ${FLINK_DIR}/examples/batch/WordCount.jar ${ARGS}
+${fetch_complete_result[@]}
```

Comment:
what is this doing?

Reply:
Added a comment `# Fetches result from AWS s3 to the OUTPUT_PATH, no-op for other filesystems and minio-based tests`